funsor/gaussian.py

```diff
@@ -26,10 +28,14 @@ def _issubshape(subshape, supershape):
     return True
 
 
-def _log_det_tril(x):
+def _log_det_tri(x):
```

Comment:
It turns out some of our matrices are upper triangular, but that's ok since these functions work with both upper- and lower- triangular matrices.